cmd/kubelet/app/server.go

```diff
@@ -201,14 +201,14 @@ func getRemoteKubeletConfig(s *options.KubeletServer, kubeDeps *kubelet.KubeletD
 				return nil, err
 			}
 			// look for kubelet-<node-name> configmap from "kube-system"
-			configmap, err := kubeClient.CoreV1Client.ConfigMaps("kube-system").Get(fmt.Sprintf("kubelet-%s", nodename), metav1.GetOptions{})
+			configmap, err := kubeClient.CoreV1Client.ConfigMaps("kube-system").Get(fmt.Sprintf("%s-%s",componentKubelet, nodename), metav1.GetOptions{})
```

Comment:
If so, maybe "kube-system" can also be replace by NamespaceSystem(pkg/api/types.go)